about/list_of_features.rst

```diff
@@ -0,0 +1,510 @@
+.. _doc_list_of_features:
+
+List of features
+================
+
+This page aims to be a synthetic list of all features currently supported by Godot.
+
+.. note::
+
+    This page lists features supported by the current development version of
+    Godot (4.0.dev). Some of these features may not be available in the current
+    stable release series (3.2).
+
+Features
+--------
+
+Platforms
+^^^^^^^^^
+
+**Can run both the editor and exported projects:**
+
+- Windows 7 and later (64-bit and 32-bit).
+- macOS 10.12 and later (64-bit).
+- Linux (64-bit and 32-bit, x86 and ARM).
+   - Binaries are statically linked and can run on any distribution if compiled
+     on an old enough base distribution.
+   - Official binaries are compiled on Ubuntu 14.04.
+
+**Runs exported projects:**
+
+- Android 4.4 and later.
+- iOS 10.0 and later.
+- HTML5 via WebAssembly (Firefox, Chrome, Edge, Opera).
+- :ref:`Consoles <doc_consoles>`.
+
+Godot aims to be as platform-independent as possible and can be ported to new
+platforms with relative ease.
+
+Editor
+^^^^^^
+
+**Features:**
+
+- Scene tree editor.
+- Script editor.
+- Support for :ref:`external script editors <doc_external_editor>` such as
+  Visual Studio Code or Vim.
+- GDScript :ref:`debugger <doc_debugger_panel>`.
+   - No support for debugging in threads yet.
+- Visual profiler with CPU and GPU time indications.
+- Performance monitoring tools.
+- Live script reloading.
+- Live scene editing.
+   - Changes will reflect in the editor and will be kept after closing the running project.
+- Remote inspector.
+   - Changes won't reflect in the editor and won't be kept after closing the running project.
+- Live camera replication.
+   - Move the in-editor camera and see the result in the running project.
+- Use the editor in dozens of languages contributed by the community.
+
+**Plugins:**
+
+- Editor plugins can be downloaded from the
+  :ref:`asset library <doc_what_is_assetlib>` to extend editor functionality.
```

Comment:
In addition to the asset library hosting plugins mentioned here, there should be a mention somewhere about the asset library's ability to host full projects, including the ability to download them via the project manager.